astropy/units/utils.py

```diff
@@ -209,8 +209,10 @@ def maybe_simple_fraction(p, max_denominator=100):
 
     The algorithm is that of `fractions.Fraction.limit_denominator`, but
     sped up by not creating a fraction to start with.
+
+    If the input is already an integer or `fractions.Fraction`, just return it.
```

Comment:
Looks like `0.0` is also returned directly, so let's mention it too while we're in there ?